src/Room.ts

```diff
@@ -264,6 +267,26 @@ export default class Room extends EventEmitter {
 		return this.peers.items.filter((p) => p !== excludePeer);
 	}
 
+	@skipIfClosed
+	public escapeMeeting() {
```

Comment:
This should be simplified down to:
```
if (!this.peers.items.some((peer) => !peer.escapeMeeting))
    this.close();
```

Reply:
We may want to send the `escapeMeeting` notification to show some notification on the client, and then forcibly close the connection.

Reply:
I think it is good now